app/src/main/java/com/example/music/player/view/songs/SongsFragment.kt

```diff
@@ -98,6 +112,7 @@ class SongsFragment private constructor() : SongsView, BaseFragment() {
         super.onStop()
         presenter.detachView()
         compositeDisposable.clear()
+        presenter.releaseMediaPlayer()
```

Comment:
Yout player should not be stopped when you are detaching fragment

Reply:
You can mark this with TODO, and remove this when you will create mechanism to stop music after closing application